src/TokenWrapper.sol

```diff
@@ -79,21 +137,21 @@ contract TokenWrapper is ITokenWrapper, ERC20 {
     /**
      * @notice Wraps the underlying token `value` and mints wrapper tokens into `msg.sender`'s balance.
      * @dev The `totalSupply` of the wrapped token MUST be less than or equal to the underlying token balance of the current contract.
-     * @param value The value to be wrapped.
+     * @param value The `wei` value to be wrapped.
      */
     function wrap(uint256 value) external override operator {
         doWrap(msg.sender, value);
     }
 
     /**
      * @dev Wraps the underlying token `value` and mints wrapper tokens into `msg.sender`'s balance.
-     * @param value The value to be wrapped.
+     * @param value The `wei` value to be wrapped.
      */
     function doWrap(address gal, uint256 value) private {
-        // Normalize the amount to have 18 decimals. We assume that `token` has 0 decimals.
-        uint256 wad = value.mul(WAD);
+        // Normalizes the amount to have 18 decimals. Assumes that `token` has 0 decimals.
+        uint256 wad = DSMathCustom.wad(value);
```

Comment:
`value * WAD` is subject to overflow. `value.mul(WAD)` was safe, but since all math functions were extracted into a library, it could either become `DSMathCustom.mul(WAD, DSMathCustom.WAD)` the above.

There is a slight gas overhead when writing the function, but personally it reads better.